src/language-server/client.ts

```diff
@@ -71,75 +71,83 @@ export class LanguageServerClient extends EventEmitter {
     return this.diagnostics.get(uri) ?? [];
   }
 
+  private initializePromise: Promise<void> | undefined;
+
+  /**
+   * Initialize or wait for in-progress initialization.
+   */
```

Comment:
Hmm, this init change wasn't intended. I just meant to request markdown.

The change is from the apidocs branch but not worth merging yet.